net-mail/automx2/automx2-2022.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
8? :)

Reply:
I haven't yet written any ebuild with V8. However, if you think it works, feel free to bump the version.